packages/roosterjs-editor-dom/lib/list/VList.ts

```diff
@@ -376,6 +376,23 @@ export default class VList {
             }
         });
     }
+
+    /**
+     * Get the index of the List Item in the current List
+     * @param input List item to find in the root list
+     */
+    getListItemIndex(input: Node) {
```

Comment:
For a splitted list e.g. `<ol start=3>`, this will return 0 for actual "3".

Need to leverage getStart(), and you just need to check if the given list node is the first one of this.items. Try not access childNodes directly.

Reply:
This wont be needed anymore as we are only going to check if the first item of the vList is impacted by the indent operation. Removing it.